docs/guides/hedgehog-engine/rangers/Model Imports/ModelIMP.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Model Importing guide for Sonic Frontiers
+---
+# Model Importing
+
+!!! info
+    This guide uses the following tools:
+
+    - [HedgeArcPack](/tools/hedgehog-engine/common/files/#hedgearcpack){ target="_blank"}
+    -Blender
+    -ModelConverter.exe
+    -ModelFBX
+    -.modelImporter add-on for Blender
+
+# Extracting Models
+
+### Creating and Extracting the Pac File
+Firstly, create a Pac file named after the character and proceed to extract the contents of the pack utilizing the HedgeArcPack tool.
+### Converting to FBX and Importing into Blender
+Next, convert the `chr_character.model` file into the FBX format using the ModelFBX tool. Please note that certain models may not necessitate conversion into FBX format. 
+If you don't need to use the FBX format, use the HedgeNeedle tool, extract the various levels of detail from the `.model` file, and rename `chr_character.0.model` to `chr_character`. 

```

Comment:
HedgeNeedle isn't listed in the necessary tools at the start of the guide. Even if optional, please list it there (you can add an "(optional)" note after its name though, with a small explanation of when/why it may be needed.